packages/@react-spectrum/s2/src/TableView.tsx

```diff
@@ -510,7 +510,9 @@ export interface ColumnProps extends RACColumnProps {
    */
   align?: 'start' | 'center' | 'end',
   /** The content to render as the column header. */
-  children: ReactNode
+  children: ReactNode,
+  /** Menu fragment to be rendered inside the column header's menu. */
+  UNSTABLE_menuItems?: ReactNode
```

Comment:
This is probably ok since users could use Collection to pass a dynamic menuItems renderer and I figure menus will be pretty small for the header menu, but this only accepting ReactNode and not a render function as well gave me pause

Reply:
Yeah, if they want to make it dynamic, then they can use a Collection, but I wasn't planning on composing to expose the render props since most s2 components dont' do that